test/given_points_training_test.jl

```diff
@@ -0,0 +1,70 @@
+
+using OrdinaryDiffEq, OptimizationPolyalgorithms, Lux, OptimizationOptimJL, Test, Statistics, Plots, Optimisers
+
+function fu(u, p, t)
```

Comment:
This file also should be renamed `NNODE_tstops_test.jl`

Reply:
Fixed